### PR TITLE
Update various Windows tools to their latest versions

### PR DESCRIPTION
**Pull request by ageis**
Git → 2.26.1
vswhere → 2.8.4
nuget → 5.5.1
aria2 → 1.35.0
ninja → 1.10.0
PowerShell → 6.2.4

### CODE REVIEW

**Comment by NancyLi1013** [comment]
Hi @ageis 
Thanks for this PR.
Could you please resolve the conflicts?
And also sign [CLA ](https://cla.opensource.microsoft.com/microsoft/vcpkg?pullRequest=10866)first?

File: scripts/vcpkgTools.xml [hunk 1]
```diff
@@ -29,11 +29,11 @@
         <archiveName>cmake-3.12.4-FreeBSD-x86_64.tar.gz</archiveName>
     </tool>
     <tool name="git" os="windows">
-        <version>2.25.1-1</version>
+        <version>2.26.1-1</version>
         <exeRelativePath>mingw32\bin\git.exe</exeRelativePath>
         <url>https://github.com/git-for-windows/git/releases/download/v2.25.1.windows.1/PortableGit-2.25.1-32-bit.7z.exe</url>
```
Comment (NancyLi1013) [inline scripts/vcpkgTools.xml:34]: It seems that the URL is not updated. 
The version should be 2.26.1 instead of 2.26.1-1.
Could you please update this?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: scripts/vcpkgTools.xml (hunk 1 above)
Comment (ageis) [reply scripts/vcpkgTools.xml:34]: thanks @NancyLi1013, just signed CLA and will address

**Comment by NancyLi1013** [comment]
Since git has been updated to 2.6.2.
Could you please resolve the conflicts?

**Comment by NancyLi1013** [comment]
@ageis 
Is work still being done for this PR?

**Comment by NancyLi1013** [comment]
@ageis 

Thanks for the PR; we're closing this for now since there's been no response. If you'd like to continue working on it, please reopen and ping us!

**Pull request closed without merging**